src/Mod/Sketcher/Gui/CommandSketcherBSpline.cpp

```diff
@@ -646,45 +646,72 @@ void CmdSketcherIncreaseKnotMultiplicity::activated(int iMsg)
     getIdsFromName(SubNames[0], Obj, GeoId, PosId);
 
     if (isSimpleVertex(Obj, GeoId, PosId)) {
-        const std::vector<Sketcher::Constraint *> &vals = Obj->Constraints.getValues();
+        int splineGeoId;
+        int knotIndexOCC;
 
-        for (std::vector<Sketcher::Constraint *>::const_iterator it= vals.begin(); it != vals.end(); ++it) {
-            if ((*it)->Type == Sketcher::InternalAlignment
-                && (*it)->First == GeoId
-                && (*it)->AlignmentType == Sketcher::BSplineKnotPoint)
+        for (auto const constraint : Obj->Constraints.getValues())
```

Comment:
Please add braces to this for.

We omit braces generally only when it is a single line within the scope of the for. This is way longer and it is difficult to read where it is intended to end.